draft-irtf-cfrg-frost.md

```diff
@@ -801,13 +801,47 @@ signature share using DeserializeElement. If validation fails, the Coordinator M
 the protocol. If validation succeeds, the Coordinator then verifies the set of
 signature shares using the following procedure.
 
-## Signature Share Verification and Aggregation {#frost-aggregation}
+## Signature Share Aggregation {#frost-aggregation}
 
 After participants perform round two and send their signature shares to the Coordinator,
-the Coordinator verifies each signature share for correctness. In particular,
-for each participant, the Coordinator uses commitment pairs generated during round
-one and the signature share generated during round two, along with other group
-parameters, to check that the signature share is valid using the following procedure.
+the Coordinator can aggregate each share to produce a final signature. This is done
+using the following procedure.
+
+~~~
+  Inputs:
+  - group_commitment, the group commitment returned by compute_group_commitment,
+    an Element in G.
+  - sig_shares, a set of signature shares z_i, Scalar values, for each participant,
+    of length NUM_PARTICIPANTS, where MIN_PARTICIPANTS <= NUM_PARTICIPANTS <= MAX_PARTICIPANTS.
+
+  Outputs: (R, z), a Schnorr signature consisting of an Element R and Scalar z.
+
+  def aggregate(group_commitment, sig_shares):
+    z = 0
+    for z_i in sig_shares:
+      z = z + z_i
+    return (group_commitment, z)
+~~~
+
+The output signature (R, z) from the aggregation step MUST be encoded as follows
+(using notation from {{Section 3 of TLS}}):
+
+~~~
+  struct {
+    opaque R_encoded[Ne];
+    opaque z_encoded[Ns];
+  } Signature;
+~~~
+
+Where Signature.R_encoded is `G.SerializeElement(R)` and Signature.z_encoded is
+`G.SerializeScalar(z)`.
+
+The Coordinator SHOULD verify this signature using the group public key before publishing or
```

Comment:
Point to verification algorithms in the appendix.

Reply:
This should point to the ciphersuite-specific verification routine, not the appendix, so I did that.